api/database/migrations/2022_08_11_161706_submitted_at_field_added_application.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class SubmittedAtFieldAddedApplication extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('pool_candidates', function (Blueprint $table) {
+            $table->timestamp('submitted_at')->nullable();
```

Comment:
Should this have a default of `null` so we don't accidentally create a pool candidate with this set? Or is it implied it is `null`? 

This is more of a question than a suggestion because I'm not sure 🤷

Reply:
It shouldn't be accidentally set anywhere, just empty/null I think
hm, let us be cautious 
@petertgiles please enlighten us

Reply:
If it's nullable and there's no default, then yes it will be null for new rows.  That said, I'm all for making implied things more explicit.

Reply:
and done
https://github.com/GCTC-NTGC/gc-digital-talent/pull/3662/commits/f86eef5e4d07ed28fc731a03b0731943aeea7278